### DISCUSSION TITLE
Building a Security Data Lake

### DISCUSSION THREAD

**Discussion by elongl** [ocsf/ocsf-docs#49] [Slack Workspace Requests]
Hi, I'm building a security data lake. I'm saving the data in the OCSF standard and I'd love to join the workspace to discuss some ideas.

Thanks.

**Comment by pagbabian-splunk** [comment]
Thank you Elon - please send over your preferred email address so that I can send you a Slack Workspace invitation.

**Reply by elongl** [reply]
Thank you very much @pagbabian-splunk.
Here's my email address: elongliks@gmail.com
I'd appreciate it if you also invite my co-founder: organy110@gmail.com

**Reply by pagbabian-splunk** [reply]
I sent you invitations, please introduce yourselves in the #general channel when you join.
